### ISSUE TITLE
Code sample in ADVANCED: @"HI, GRANDMA!" should be sentence case

### ISSUE THREAD

**Issue by kenmhaggerty** [learn-co-curriculum/objc-speaking-grandma#6]
In the code example under the ADVANCED section,

`NSString *talkToGrandma = @"HI, GRANDMA!";
NSString *shoutAtGrandma = [talkToGrandma uppercaseString];`

should be:

`NSString *talkToGrandma = @"Hi, Grandma!";
NSString *shoutAtGrandma = [talkToGrandma uppercaseString];`

**Comment by markedwardmurray** [comment]
That would miss the point of using the `BOOL` to capture the state of the code before changing it. This section is show that booleans reflect the result of a calculation made at the point in time in which is assigned, not when they are evaluated.

**Comment by kenmhaggerty** [comment]
Right, but performing method `uppercaseString` on `talkToGrandma` in line 2 of the code example under Advnaced does nothing when `talkToGrandma` is set to `@"HI, GRANDMA!"` in line 1? In the end it doesn't affect the outcome of either if/else statement.

**Comment by marksquad** [comment]
The point of uppercasing the string and saving it as `shoutAtGrandma` is how the `isEqualToString:` method is applied to detect if `talkToGrandma` is an uppercase string. If uppercasing the `talkToGrandma` string does nothing then it means the string is already uppercased and should pass the check.
